src/resources/appointment.ts

```diff
@@ -58,20 +58,17 @@ export interface AppointmentParameters {
     };
     type: string;
   };
-  meta?: {
-    booker?: number;
-    notify?: {
-      client?: boolean;
-      user?: boolean;
-    };
-    utm?: {
-      campaign?: string;
-      content?: string;
-      source?: string;
-      medium?: string;
-      term?: string;
-    };
+  meta?: AppointmentMeta;
+}
+
+export interface AppointmentMeta {
```

Comment:
I'm not certain this was necessary to extract. I see you're using it in the method below, but all that method really needs is a type like this in my opinion.

```ts
{ [key: string]: string | number | boolean }
```

Reply:
it was extracted already but I added to it. Do you want the interface removed entirely or just put back? It had `booker` in it originally.

Reply:
It looks to me like you added that interface in this PR, didn't you? I think it'd be easy enough to just leave it the way it was and then add the dynamic `[key: string]` part inside of the `meta` object. Kinda something like this:

```ts
export interface AppointmentParameters {
  ...
  meta?: {
    ...
    [key: string]: string | number | boolean;
  };
}
```

Reply:
I think there's just the appearance I added it caused I moved it up a bunch of lines from where it was originally :p
I like your suggestion, I'll do that wherever the interface is being using and get rid of it